spark/src/main/java/org/apache/iceberg/actions/Actions.java

```diff
@@ -88,6 +89,10 @@ public ExpireSnapshotsAction expireSnapshots() {
     return new ExpireSnapshotsAction(delegate);
   }
 
+  public BaseRemoveFilesSparkAction removeFilesAction() {
```

Comment:
Let's not add it to the old actions API.

Reply:
Let's add it to `ActionsProvider` interface instead.

Reply:
Should we move the `org.apache.iceberg.spark.actions.SparkActions` from `spark2` to `spark` module, so that it is accessible to test in `spark` module?